modules/ingestor/src/graph/organization.rs

```diff
@@ -40,10 +41,24 @@ impl<'g> OrganizationContext<'g> {
     }
 }
 
-impl super::Graph {
+impl Graph {
+    #[instrument(skip(self, tx), err)]
+    pub async fn get_organizations(
```

Comment:
related: I'd still argue we should be deleting get_plurals() and defer that ish to the appropriate fundamental Service.

Reply:
Created a tracking issue: https://github.com/trustification/trustify/issues/547